src/redist/targets/GenerateMSIs.targets

```diff
@@ -177,6 +181,21 @@
                     '$(VSToolsNupkgFile)'" />
   </Target>
 
+  <Target Name="GenerateVSToolsResolverNupkg"
```

Comment:
Is there anything that ensures that SdkResolver.csproj gets built before redist.csproj?  I think you might need a ProjectReference between the two.

Reply:
This is only called  by pack phase. SdkResolver is built in earlier build phase.

Reply:
It seems like this could break if you call `dotnet pack` on redist.csproj instead of going through the top-level build script.

Reply:
I see.

Reply:
Sigh. dotnet pack work already. We end up not building the core-sdk-tasks. There's code that tries to do this, but it's weird: before check invalid configuration target and I can't figure out why that isn't run in this case so far. In general, redist is very cute in how it orchestrates things. Related to my earlier comment, this should be cleaned up. Interpreting the binlog at the moment is extremely frustrating.

Reply:
Yeah, the cuteness with core-sdk-tasks is my fault.  The goal was to let you have the tasks project loaded in the same solution as the rest of the code, without having build failures because of locked task DLLs.  So it tries to build the core-sdk-tasks to a separate output folder which is where they're actually loaded from in the build.